docs/index.md

```diff
@@ -450,7 +450,7 @@ Decision-making often results in a narrowly-scoped "Pass" that embodies conclusi
 
 The following tools are helpful to validate Health Card artifacts:
 
-* The [HL7 FHIR Validator](https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator) can be used to validate the Health Card's FHIR bundle
+* The [HL7 FHIR Validator](https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator) can be used to validate the Health Card's FHIR bundle. It can load the smart card from the bundle directly, or from the jws, or from a text file starting with shc:/ (but not - yet - the QR code itself)
```

Comment:
```suggestion
* The [HL7 FHIR Validator](https://confluence.hl7.org/display/FHIR/Using+the+FHIR+Validator) can be used to validate the Health Card's FHIR bundle. It can load the smart card from the bundle directly, or from the jws, or from a text file starting with `shc:/` (but not a QR image).
```

I'm not 100% sure what loading from the bundle directly means. Also, do you process `.smart-health-card` JSON files?